webdriver/tests/bidi/browsing_context/reload/wait.py

```diff
@@ -0,0 +1,192 @@
+import asyncio
+import pytest
+
+pytestmark = pytest.mark.asyncio
+
+
+@pytest.mark.parametrize("wait", ["none", "interactive", "complete"])
+async def test_expected_url(bidi_session, inline, new_tab, wait):
+    url = inline("<div>foo</div>")
+    await bidi_session.browsing_context.navigate(context=new_tab["context"],
+                                                 url=url,
+                                                 wait=wait)
+    result = await bidi_session.browsing_context.reload(
+        context=new_tab["context"], wait=wait)
+    assert result == {}
+    if wait != "none":
+        contexts = await bidi_session.browsing_context.get_tree(
+            root=new_tab["context"], max_depth=0)
+        assert contexts[0]["url"] == url
+
+
+@pytest.mark.parametrize(
+    "wait, expect_timeout",
+    [
+        ("none", False),
+        ("interactive", False),
+        ("complete", True),
+    ],
+)
+async def test_slow_image_blocks_load(bidi_session, inline, new_tab, wait,
+                                      expect_timeout):
+
+    script_url = "/webdriver/tests/bidi/browsing_context/navigate/support/empty.svg"
+    url = inline(f"<img src='{script_url}?pipe=trickle(d10)'>")
+
+    await bidi_session.browsing_context.navigate(context=new_tab["context"],
+                                                 url=url,
+                                                 wait=wait)
```

Comment:
Similar comment for all the other tests in this file.

This navigate call will take 10 seconds to resolve for the "complete" case, and overall the test will easily go over the 45 seconds timeout. 

Ideally we need a server which would only start slowing down resources for the reload call. I can't really think of an easy way to do this, so maybe in a first step we reduce the trickle(d10) to d3 and add the long timeout metadata for the file?

Reply:
Those navigate should also probably always use wait="complete". The parametrized `wait` should only be applied to reload calls (although it could be interesting to test reloads starting in the middle of an existing navigation, but that would probably be a different test then)

Reply:
In Marionette tests for Firefox we have HTML that looks like `<img src="/slow?delay=4" id="slow" />`. So with wptserve we could use tickle as well for slowly loading the resource only, right?

Reply:
Oh that's right. Yes, so I don't think that this is easy right now. We might have something like that later on when we can emulate network throughput and then slow down the transfer of page data. But right now I don't see an option.

Reply:
Done